ScattReport.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Scatt.java
+ *  
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ * @author Michelle Melton
+ * @version Mar 2017
+ */
+
+/**
+ * Scratch Code Analysis Tool for Teachers (SCATT). 
+ * 
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ * @author Michelle Melton
+ * @version Mar 2017
+ */
+public class ScattReport
+{
+    /**
+     * Main method for ScattReport class.
+     *
+     * @param args 
+     */
+    public static void main(String[] args)
+    {
+    }
+
+    /**
+     * Method for makeing the report.
+     */
+    public static void makeReport()
```

Comment:
I'm not sure that this should be a static method. Won't we create a Report object in Scatt.java? Or will we just call this method and within it contain calls to the Submission objects functions to get the data on each submission? How are you envisioning the design implementation for this? @KaraBeason what do you think?